plugins/woocommerce/src/Blocks/Assets/Api.php

```diff
@@ -216,6 +226,7 @@ public function get_script_data( $relative_src, $dependencies = [] ) {
 				'version'      => ! empty( $asset['version'] ) ? $asset['version'] : $this->get_file_version( $relative_src ),
 				'dependencies' => ! empty( $asset['dependencies'] ) ? $asset['dependencies'] : [],
 			);
+			$this->script_data_modified = true;
```

Comment:
There is a linting error here:

```suggestion
			$this->script_data_modified         = true;
```